app/locales/de.json

```diff
@@ -79,28 +79,64 @@
     {
       "type": "chart",
       "id": "section_4",
-      "title": "Erhebung 4",
-      "disclaimer": "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Assumenda soluta sed consequatur rem saepe dolores voluptates recusandae explicabo officiis",
+      "title": "Welche Innovatoren sind wichtig für Berlin/Deutschland?",
+      "disclaimer": "Berlin entwickelt sich zunehmend zum Software-Standort, während bundesweit weiterhin der Maschinen- und Fahrzeugbau dominiert.",
       "charts": [
         {
           "id": "growth_bubble",
-          "description": "Ein funky chart für <strong>Berlin</strong> bleibt hingegen weiterhin ungeschlagen die Pharmaziebranche. Keine andere Branche ist derart aktiv in der deutschen Forschungslandschaft.In den hier nicht weiter aufgeschlüsselten restlichen Branchen sind vor allem die Bereiche der Medien und der Finanzdienstleistungen verantwortlich für den moderaten Anstieg zwischen den Jahrzehnten. <br><br>Im <strong>bundesweiten</strong> Vergleich dominiert weiterhin ganz klar der Maschinen- und Fahrzeugbau im Bereich der Industrieforschung. Aber auch hier konnte die Softwarebranche ihre Anteile signifikant ausbauen.",
-          "toggleText": "Anteile der Industrieforschung in "
+          "description": "Im Gegensatz zur Bundesrepublik liegt in <strong>Berlin</strong> ein großer Schwerpunkt auf den Unternehmen der Software und Datenverarbeitung, diese Branche umfasst die größte Anzahl der Beschäftigten und kann den größten Wachstum vorweisen.<br><br>Mit weniger Wachstum, bleibt trotzdem weiterhin die Branche der Pharma-, Chemie- und Kunststoffunternehmen Spitzenreiter im Bereich der FuE-Ausgaben. <strong>Bundesweit</strong> hingegen bleibt wenig überraschend die Branchen des Maschinen- und Fahrzeugbaus klar vorn, aber auch die Branchen Elektroindustrie und Instrumententechnik, sowie die Branche der Pharma-, Chemie- und Kunststoffverarbeitenden Betriebe. ",
+          "toggleText": "Welche Innovatoren sind besonders wichtig für "
```

Comment:
`die größte Anzahl der Beschäftigten` ist meines Wissens nach nicht korrekt.

Entweder `die größte Anzahl an Beschäftigten`, oder `die größte Zahl der Beschäftigten` aber eventuell sogar eher `der größte Anteil an Beschäftigten`, oder?

Reply:
Vielleicht FuE Ausschreiben?

Reply:
`hingegen bleibt wenig überraschend die Branchen` -> Plural also `bleiben`

Reply:
`Pharma-, Chemie- und Kunststoffverarbeitenden Betriebe` -> `Pharma-, Chemie- und Kunststoffverarbeitenden-Betriebe`?

Reply:
`die größte Anzahl der Beschäftigten` - glaube ich ist auch richtig, aber "an" passt besser mit dem darauffolgenden Halbsatz zusammen, daher so übernommen.

`FuE` - ich wollte es halt konsequent halten, habe aber mal das Dokument so umgebaut, dass die lange, ausgeschrieben Form bevorzugt wird, aber in Ausnahmefälle wie in Überschriften oder wenn es zu Wortungetümen führt ("_Forschungs- und Entwicklungs- oder anderen Innovationsaktivitäten_") bei FuE belassen. 

`bleibt/bleiben` - merci!

` Kunststoffverarbeitende Betriebe` - ist in dem Fall ein Adjektiv und daher ist mWn. der Bindestrich falsch,